wondrous-app/graphql/queries/task.ts

```diff
@@ -11,6 +11,15 @@ export const GET_TASK_BY_ID = gql`
   ${TaskFragment}
 `;
 
+export const GET_TASK_TEMPLATES_BY_ID = gql`
```

Comment:
if we do user_id we should call it GET_TASK_TEMPLATES_BY_USER_ID for more clarify